contrib/python/api/skydive/rest/client.py

```diff
@@ -65,11 +71,55 @@ def lookup(self, gremlin, klass=None):
                                                  context=context))
 
         opener = request.build_opener(*handlers)
-        req = request.Request(url,
-                              data.encode(),
-                              {'Content-Type': 'application/json'})
 
+        req = request.Request(url, data=urlencoder.urlencode(data).encode())
         resp = opener.open(req)
+        if resp.getcode() == 200:
```

Comment:
Why not handle set the authenticate to True here? Why leave it up to the code using this code?